### ISSUE TITLE
Weapon crate support for new loadouts

### ISSUE THREAD

**Issue by AJCStriker** [23rd-gaming-division/OperationCrimsonSunrise.Panthera3#6]
@Oges has put together some new load outs for the missions which are stored in the Platoon Leaders Book. We need to get them cleared for use by him and put in.

We need to work out the camo we are going to use for Panthera and put together a arsenal box like we used to have for each role.

Further we should work out an arsenal box that allows for certain options in terms of glasses, helmet type ( with goggles / cameras ) and a few scope options ( CompM4s, ACOGs and EOTechs ).

Possible to pull @mrgwilliam loadout scripts from the training server?

**Comment by rasolercrespo** [comment]
I also have a loadout script we used in the Op before Oil Barron. I made it for loadout boxes and works similar to the one from Rodriguez.

**Comment by Oges03** [comment]
The loadout is approved.

**Comment by mrgwilliam** [comment]
It is important to note that those loadouts are not 100% compatible with the recent modpack updates. They will need to be updated to be fully compatible and CSE ready.

**Comment by AJCStriker** [comment]
It’s a good point. We should try and get standard loudest issued that contain the required medical kits, as well as possibly some CSE tablets so we can use Blue Force Tracker.=

**Comment by rasolercrespo** [comment]
@AJCStriker and @mrgwilliam; I found the scripts I referenced. Let me know where I can access said loadouts. I will work on the Arsenal boxes meanwhile.

**Comment by AJCStriker** [comment]
https://docs.google.com/spreadsheets/d/1am-sUQV55SIMA7Dzaea3WGwrwVeEgGPWCkbERA9O_xU/pubhtml?gid=1938942237&single=true

These loadouts still require medical equipment and CSE BLUFOR tracker equipment

**Comment by Oges03** [comment]
I will update the loadout scripts with the appropriate medical equipment and CSE by wednesday

**Comment by Oges03** [comment]
I've added medical supplies and rations from the CSE system, I'm not seeing anything that looks like BFT equipment unless you mean the C2 stuff

**Comment by Oges03** [comment]
The Spartan Platoon Standard Loadouts will be posted in the Platoon Leader Book tonight (152359ZJAN15).  That is the kit that we should be carrying into battle.  SLs will be instructed to ensure that all personnel have every SOP loadout saved and we will spot check the kit prior to the operation to make sure everyone has what they need.

**Comment by Oges03** [comment]
I've noted some issues with the loadout scripts in the platoon leader book and the import functions. When the loads are imported via Armory, some things are missing.  I've tried a few times to make it work with no success.  I don't think it will affect scripting into the start loads, only if they are imported through armory.  I will publish the PL and PSG loadouts as well since I noticed they weren't in the sheet
